### ISSUE TITLE
@Api description not visible in swagger.json file

### ISSUE THREAD

**Issue by poseidon1011** [kongchen/swagger-maven-plugin#205]
When I give some description in @Api annotation like
@Api(value = "UsersController", description = "Controller for users"

that information is not passed to output swagger.json file

"tags" : [ {
    "name" : "UsersController", description="[there should be: Controller for users]"
  },

instead, the whole description part is missing

I've seen that you read that information in SpringMvcApiReader.java -> read() method, but nothing happens with that text

Is it possible to handle this information and pass it to json file? Swagger-ui can read that information from file and display near the controller's name if I enter it manually

Using smp v.3.0.1

**Comment by alexkrishnan** [comment]
I would love to see this fixed as well, the top level API descriptions are the most important part of documentation! Poseidon, have you found a workaround other than manually editing the json files?

**Comment by poseidon1011** [comment]
No, just assumed that I dont have that description for now. I'm writting good descriptions hoping to have the functionality in the future.

**Comment by suchita-zade** [comment]
Is there any work-around or fix for this issue. This is working fine for Jax-RS (as  it using JaxrsReader.java).
Can someone add the same code in SpringMvcApiReader.java and fix this issue for Springboot Project?

**Comment by bratwurzt** [comment]
This PR adds Api description to a Tag, if there's a single tag defined. If there are no explicit tags, existing code already adds one.
